mappings/net/minecraft/screen/slot/ItemCombinationSlotManager.mapping

```diff
@@ -0,0 +1,33 @@
+CLASS net/minecraft/unmapped/C_vkmtnvmw net/minecraft/screen/slot/ItemCombinationSlotManager
+	FIELD f_fagsmfxk inputSlots Ljava/util/List;
```

Comment:
wait this is about recipe stuff, right? shouldn't `input` be called `ingredient` then?

Reply:
yes, I'm entrenched in my `input` ways
will fix!